okhttp/third_party/okhttp/java/io/grpc/okhttp/internal/Platform.java

```diff
@@ -130,6 +135,29 @@ public void connectSocket(Socket socket, InetSocketAddress address,
     socket.connect(address, connectTimeout);
   }
 
+  public static SSLContext getSslContext() throws NoSuchAlgorithmException {
```

Comment:
This file was supposed to be a copy of OkHTTP.  Is this function upstreamed in OkHTTP?  If not, it should probably go somewhere else.